src/mkslides/markupgenerator.py

```diff
@@ -328,6 +331,14 @@ def __copy_theme(
 
         return relative_theme_path
 
+    def __copy_static(self, static: dict[str, str]):
+        for src, dst in static.items():
+            src_path = Path(src).resolve(strict=True)
+            logger.info(f"Copying static '{src_path.absolute()}'")
```

Comment:
Redundant logging, the low level copy functions log it anyway.

Reply:
Removed 👍🏼